common/components/WalletDecrypt/LedgerNano.tsx

```diff
@@ -65,9 +86,12 @@ export default class LedgerNanoSDecrypt extends Component<Props, State> {
             </a>
           </div>
         </div>
-
-        <div className={`LedgerDecrypt-error alert alert-danger ${showErr}`}>{error || '-'}</div>
-
+        {error && (
+          <p className="LedgerDecrypt-error alert alert-danger is-showing">
+            <strong>Error: </strong>
+            {error}
+          </p>
+        )}
```

Comment:
Not saying to change this, but just wanted to make sure it was clear that this was always rendered before, and is-showing was added so that the content didn't jump quite as much when an error popped up. It always filled the area it would in the content.

Reply:
I was actually just messing around with it and forgot to remove it before committing. I'll make sure to revert it